lib/internal/repl/utils.js

```diff
@@ -394,7 +394,7 @@ function setupPreview(repl, contextSymbol, bufferSymbol, active) {
     }
 
     const inputPreviewCallback = (error, inspected) => {
-      if (inspected === null) {
+        if (inspected == null) {
```

Comment:
Maybe even `if (!inspected)` would be clearer but I wanted to make minimal changes that worked

Reply:
`== null` is perfectly fine, we want to filter nullish values, not falsy ones :)